### PR TITLE
Arm clarification added

### PR DESCRIPTION
**Pull request by Abhi464**
Resolves issue #11618

### CODE REVIEW

**Comment by codebytere** [comment]
@Abhi464 can you rebase this on master? it's throwing some weird threading errors

**Comment by codebytere** [comment]
Failures are resultant of a DCHECK error that will be fixed in https://github.com/electron/electron/pull/11697

**Comment by codebytere** [comment]
Please rebase this on `master` now that https://github.com/electron/electron/pull/11697 has been merged and we'll get this merged!

File: docs/tutorial/supported-platforms.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ The prebuilt `ia32` (`i686`) and `x64` (`amd64`) binaries of Electron are built
 Ubuntu 12.04, the `arm` binary is built against ARM v7 with hard-float ABI and
 NEON for Debian Wheezy.
 
+Here `arm` architecture means for both `arm` and `armv7L` as they are just two
```
Comment (lurch) [inline docs/tutorial/supported-platforms.md]: IMHO the 'L' in `armv7l` should be lowercase, to match the actual filenames ;)

**Comment by codebytere** [comment]
Sorry, this wasn't rebased in time so another PR took its place https://github.com/electron/electron/pull/11825 as it was completed more quickly.

**Pull request closed without merging**